acceptance-tests/run-tests

```diff
@@ -66,24 +65,23 @@ setArgsOverrides(
 (async () => {
   try {
     const testRunner = new Jasmine();
+    testRunner.exitOnCompletion = false;
     testRunner.loadConfigFile(path.join(__dirname, 'jasmine.json'));
     testRunner.addReporter(reporter);
-    testRunner.onComplete(function(passed) {
```

Comment:
This is deprecated. They suggested to use `testRunner.exitOnCompletion = false` instead